client/web/src/enterprise/batches/detail/changesets/ChangesetSelectRow.tsx

```diff
@@ -91,6 +93,22 @@ const AVAILABLE_ACTIONS: ChangesetListAction[] = [
             />
         ),
     },
+    {
+        type: 'merge',
+        experimental: true,
+        buttonLabel: 'Merge changesets',
+        dropdownTitle: 'Merge changesets',
+        dropdownDescription: 'Merge all selected changesets, if constraints are met.',
```

Comment:
_If constraints are met_ feels a little hazy to me. How about:

```suggestion
        dropdownDescription: 'Attempt to merge all selected changesets. Some changesets may be unmergeable if there are rules preventing merge, such as CI requirements.',
```